hazelcast/src/test/java/com/hazelcast/map/BasicMapTest.java

```diff
@@ -131,10 +131,19 @@ public void init() {
 
     protected Config getConfig() {
         Config cfg = smallInstanceConfig();
-        cfg.getMapConfig("default").setStatisticsEnabled(statisticsEnabled);
+        cfg.getMapConfig("default")
+                .setStatisticsEnabled(statisticsEnabled)
+                .setPerEntryStatsEnabled(statisticsEnabled);
```

Comment:
Maybe add some comment to the field that we intentionally use it for controlling both per-map and per-entry statistics. With this approach, we test both the presence/absence of the two stats. 

Btw, do we have any test for the `per-map:off`, `per-entry:on` configuration?

Reply:
Updated test parameter list to reflect all combinations